import java.time.LocalDate;

public class Repostaje {
    int id;
    int idCliente;
    LocalDate fecha;
    double importe;
    double litros;
    String combustible;

    public void Repostaje(int id, int idCliente, double importe, LocalDate fecha, double litros, String combustible) {
        this.id = id;
        this.idCliente = idCliente;
        this.importe = importe;
        this.fecha = fecha;
        this.litros = litros;
        this.combustible = combustible;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdCliente() {
        return idCliente;
    }

    public void setIdCliente(int idCliente) {
        this.idCliente = idCliente;
    }

    public LocalDate getFecha() {
        return fecha;
    }

    public void setFecha(LocalDate fecha) {
        this.fecha = fecha;
    }

    public double getImporte() {
        return importe;
    }

    public void setImporte(double importe) {
        this.importe = importe;
    }

    public double getLitros() {
        return litros;
    }

    public void setLitros(double litros) {
        this.litros = litros;
    }

    public String getCombustible() {
        return combustible;
    }

    public void setCombustible(String combustible) {
        this.combustible = combustible;
    }
}
